src/components/ImportExport.js

```diff
@@ -181,6 +196,47 @@ class ImportExport extends React.Component {
     }
   }
 
+  generateCSV = () => {
+    const { hydrants, trails } = this.props;
+    const hydrantsRows = [
+      ['Trail_Name', 'Hyd_Index', 'Hyd_ID', 'Hyd_State', 'Hyd_Hours',
+      'Hyd_Gun', 'Hyd_Gpm', 'Hyd_Notes', 'Hyd_Elevation',
+      'Hyd_Target_Gallons', 'Hyd_Total_Gallons', 'Hyd_Cfm', 'Hyd_Pressure_Zone']
+    ]
+    trails.keySeq().forEach((trailId) => {
+      const trailName = trails.getIn([trailId, 'features'])[0].get('originalTrailName')
```

Comment:
It doesn't look like `generateCSV` has to go through `exportFile`, so new trails won't have an `originalTrailName` assigned yet unless you exported them already, or am I off?

Reply:
Good catch. Fixed w/ commit  3b1095c